app/src/hooks/api/useObservationApi.ts

```diff
@@ -38,6 +38,15 @@ const useObservationApi = (axios: AxiosInstance) => {
       }
     );
 
+    console.log('data is ', data);
+
+    if (data.submissionId) {
+      console.log('Validating the dwca submission');
+      axios.post(`/api/dwc/validate`, {
+        occurrence_submission_id: data.submissionId
+      });
+    }
```

Comment:
I know this is a temporary thing, but I wouldnt put it inside this existing function. I'd make a new function, and then just call it in the parent, after the upload call is made.